### PR TITLE
Fixing errors related to Go 1.20

### PR DESCRIPTION
**Pull request by scottmason88**
## Description

This pull request will fix errors that came about with Go 1.20 deprecating rand.Seed.

## Changes

* Changed loop in tablegenerator.go to use the variadic function
* Removed the random number generator seed from main.go, and added a package local variable doing this to service.go, since this is the only place the seed from main.go was used.

## Reference

Issue #1775 

**Release Note**

```release-note
 Fix deprecated functions related to Go 1.20 
```

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -7,9 +7,9 @@
 // the empty line separators.
 
 ////
-| 🎁🐛🐣🗑️✨
-|
-| https://github.com/knative/client/pull/[#]
+| 🐛
+| Updated to work with Go 1.20
+| https://github.com/knative/client/pull/1775[#1775]
```
Comment (rhuss) [inline CHANGELOG.adoc:12]: Oops, I just saw that the Changelog is quite outdated 😬 (@dsimansk would it make sense to bring it up to date, covering the releases in the second half of 2022 ?)

However, @scottmason88 can you please keep the template text and move your part out of the comment (`//// .... ////`) ? Also you should add a new table for the 1.10 release (the next release), without a date yet. Then add your entry as the first row. That would be great !

**Review by rhuss — COMMENTED** [review]
Thanks, and congrats on your first client PR ! Looks great. If you could fix the Changlog entry, that would be great.

File: CHANGELOG.adoc (hunk 1 above)
Comment (dsimansk) [reply CHANGELOG.adoc:12]: Well, we don't use it anymore. I should have added a deprecation notice to that file probably. We're using release script generated release notes.

Comment (dsimansk) [reply CHANGELOG.adoc:12]: I'd skip the update from this PR.

Comment (scottmason88) [reply CHANGELOG.adoc]: I removed my changes to the changelog so that the comment is still there.

**Review by rhuss — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**